reference/procedural-proto-nodes.md

```diff
@@ -102,11 +102,20 @@ Using the [lua-gd](http://ittner.github.io/lua-gd) module it is possible to gene
 
 In addition to these fonts, it is possible to add other TrueType fonts file in your `PROJECT_HOME/fonts` directory.
 
+### Optimization
+
+Using procedural PROTOs can greatly increase the lodaing time of your worlds because every procedural PROTO need to be evaluated.
+
+To reduce the number of evaluation you can add the `static` tag as a comment in the PROTO header (i.e. `# tags: static`). If the `static` tag is present, then in a world who uses the same procedural PROTO several times, the PROTO is evaluated only once if all the fields value are the same.
+> **Note**:
+This tag should not be used if the result of the PROTOs depend of something else than the fields value (e.g. use a random value).
```

Comment:
if the result (...) depend**s**

Is it correct to use the preposition 'of' with the verb 'to depend'?